Classes/ReportsConfigurationStatus.php

```diff
@@ -58,9 +58,9 @@ public function getLangModes() {
 			'fail' => array(),
 		);
 		$value = $GLOBALS['LANG']->sL('LLL:EXT:languagevisibility/locallang_db.xml:reports.ok.value');
-		$severity = tx_reports_reports_status_Status::OK;
+		$severity = \TYPO3\CMS\Reports\Status::OK;
 
-		$rootTpls = \TYPO3\CMS\Backend\Utility\BackendUtility::getRecordsByField('sys_template', 'root', '1', '', 'pid');
```

Comment:
This new behaviour of Mysql is known for decades in Postgresql and when you think more about SELECT * + GROUP BY col, it returns inaccurate data.

If there is data:
pid uid
1    2
1    3

and you group by pid, data for uid should be explicitely defined e.g. MAX(uid), MIN(uid). Mysql somehow returned data.